tests/sentry/data_export/endpoints/test_data_export.py

```diff
@@ -46,7 +46,7 @@ def test_new_export(self):
         assert response.data == {
             "id": data_export.id,
             "user": {
-                "id": six.binary_type(self.user.id),
+                "id": six.text_type(self.user.id).encode("utf-8"),
```

Comment:
I'm not sure we actually need to `.encode` here. This usage just seems to want the user id to be some kind of string instead of an int.

Reply:
Yeah i'll come back to this. But for now this seems to be the most obvious translation that doesn't do completely the wrong things.